pkg/startup/startup.go

```diff
@@ -453,6 +453,9 @@ func configureIPsAndSubnets(node *api.Node) (bool, error) {
 			log.Warnf("Autodetection of IPv4 address failed, keeping existing value: %s", node.Spec.BGP.IPv4Address)
 			validateIP(node.Spec.BGP.IPv4Address)
 		}
+	} else if ipv4Env == "none" && node.Spec.BGP.IPv4Address != "" {
+		log.Warnf("Autodetection for IPv4 disabled, keeping existing value: %s", node.Spec.BGP.IPv4Address)
```

Comment:
I don't think this needs to be a warning. Should probably be an "Info" level log (since this is an expected code path).

Reply:
Changed to log.Infof as suggested. Same for ipv6